org.osgi.service.feature/src/org/osgi/service/feature/Feature.java

```diff
@@ -13,14 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * SPDX-License-Identifier: Apache-2.0 
+ * SPDX-License-Identifier: Apache-2.0
  *******************************************************************************/
 package org.osgi.service.feature;
 
+import org.osgi.annotation.versioning.ProviderType;
+
```

Comment:
This is another code move which the formatter will move back.

Reply:
Yes you are right - my local experimentation project has different formatting rules. I'll line them up and fix the PR with them.

Reply:
Fixed

Reply:
You should squash the commits into one commit.